docs/30_commands.md

```diff
@@ -226,5 +234,109 @@ Available flags for the command:
 - `--certificate-authority`, to provide the path to a custom CA certificate
 - `--insecure-skip-tls-verify`, to disallow the check the validity of the certificate of the remote endpoint
 - `--context`, to specify a different context from the currently selected one
-- `--company-id`, to set the ID of the desired company
-- `--project-id`, to set the ID of the desired project
+- `--company-id`, to set the ID of the desired Company
+- `--project-id`, to set the ID of the desired Project
+
+## marketplace
+
+View and manage Marketplace items
+
+All the subcommands inherit the following flags:
+
+```
+      --auth-name string               the name of the miactl auth to use
+      --certificate-authority string   path to a cert file for the certificate authority for the selected endpoint
+      --company-id string              the ID of the Company
+  -c, --config string                  path to the config file default to $HOME/miactl/config
+      --context string                 the name of the miactl context to use
+      --endpoint string                the address and port of the Mia-Platform Console server
+      --insecure-skip-tls-verify       if true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
+      --verbose                        increase the verbosity of the cli output
+```
+
+### list
+
+List Marketplace items
+
+#### Synopsis
+
+List the Marketplace items that the current user can access.
+
+```
+miactl marketplace list [flags]
+```
+
+### get
+
+Get a Marketplace item
+
+#### Synopsis
+
+Get a single Marketplace item by its ID
+
+```
+miactl marketplace get resource-id [flags]
+```
+
+### delete
+
+Delete a Marketplace item
+
+#### Synopsis
+
+Delete a single Marketplace item by its ID
+
+```
+miactl marketplace delete resource-id [flags]
+```
+
+### apply
+
+Create or update Marketplace items
+
+#### Synopsis
+
+Create or update one or more Marketplace items.
+
+The flag -f accepts either files or directories. In case of directories, it explores them recursively.
+
+Supported formats are JSON (.json files) and YAML (.yaml or .yml files).
+
+The file can contain an image object with the following format:
+"image": {
+	"localPath": "./someImage.png"
+}
+The localPath can be absolute or relative to the file location.
+The image will be uploaded along with the Marketplace item.
+Before being applied, the "image" key will be replaced with the "imageUrl" referring to the uploaded image.
+You can retrieve the updated item with the "get" command.
+
+You can also specify the "supportedByImage" in a similar fashion.
+
+Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is illegal.
```

Comment:
may opt for "is not allowed" instead

Reply:
I wanted to emphasize the fact that doing it raises an error. Maybe we can do this?
```suggestion
Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is ambiguous and raises an error .
```